script/un-s4.R

```diff
@@ -0,0 +1,117 @@
+pkgload::load_all()
+
+root <- "~/git/R/r-dbi/RSQLite"
+
+if (!exists("parsed_full")) {
+  parsed_full <- parse_package(root)
+} else {
+  message("Reusing parse data from ", dirname(dirname(parsed_full$filename[[1]])))
+}
+
+parsed_nested <-
+  parsed_full %>%
+  nest(data = -filename)
+
+# If necessary, use a slice here
+parsed <-
+  parsed_nested[, ] %>%
+  unnest(data)
+
+method_idx <- map_lgl(parsed$code, ~ {
```

Comment:
This might be made more robust by using name indexing, e.g. arg 4 is "definition", since arguments might not be fed in the standard order. match.call() can be used to make args explicit in a call.
We could test for the value of `setMethod()`rather than the symbol if we want extra robustness, to deal with `methods::setMethod()` or `alias <- setMethod` though I guess both uses are unlikely.